Zend/zend_exceptions.c

```diff
@@ -639,11 +643,13 @@ ZEND_METHOD(Exception, getTraceAsString)
 /* {{{ Return previous Throwable or NULL. */
 ZEND_METHOD(Exception, getPrevious)
 {
-	zval rv;
+	zval *prop, rv;
 
 	ZEND_PARSE_PARAMETERS_NONE();
 
-	ZVAL_COPY(return_value, GET_PROPERTY_SILENT(ZEND_THIS, ZEND_STR_PREVIOUS));
+	prop = GET_PROPERTY(ZEND_THIS, ZEND_STR_PREVIOUS);
+	ZVAL_DEREF(prop);
+	ZVAL_COPY(return_value, prop);
```

Comment:
We have ZVAL_COPY_DEREF, although I see pre-existing places with this pattern too...